decidim-core/config/locales/en.yml

```diff
@@ -1814,7 +1801,6 @@ en:
       flag_user_modal:
```

Comment:
We should also change the title (line 1824). It should be:

> title: Report inappropriate participant

Reply:
Fix in [143535c](https://github.com/decidim/decidim/pull/14031/commits/143535c542e8a07524758c2ee351c63d923a5e44)